scripts/std-lib-regression.sh

```diff
@@ -2,20 +2,26 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 
+# Deliberately not enabling this, since we expect a failure currently and are failing based on 'grep' later
```

Comment:
You could add a `| true` at the end of the command that we expect to fail for now. Then you can enable `set -eu` to ensure that other errors with is correctly handled.